### PR TITLE
cleaner startup exec in order to build the image and run the containe…

### PR DESCRIPTION
**Pull request by gnu300**
…r with podman/libpod

### CODE REVIEW

**Comment by mprasil** [comment]
For reference this change was discussed under #718 to enable running on podman with cgroupv2, but the change should be also an improvement for regular docker by making the `WORKDIR` and binary location more explicit and less reliable on defaults.

@gnu300 can you also make the same change to the rest of the Dockerfiles?

File: docker/amd64/sqlite/Dockerfile [hunk 1]
```diff
@@ -94,4 +94,5 @@ COPY docker/healthcheck.sh ./healthcheck.sh
 HEALTHCHECK --interval=30s --timeout=3s CMD sh healthcheck.sh || exit 1
 
 # Configures the startup!
-CMD ["./bitwarden_rs"]
+WORKDIR /
+CMD ["/bitwarden_rs"]
```
Comment (mprasil) [inline docker/amd64/sqlite/Dockerfile]: Change the rest of the Dockerfiles please

Comment (gnu300) [reply docker/amd64/sqlite/Dockerfile]: np -> done

Comment (mprasil) [reply docker/amd64/sqlite/Dockerfile]: Almost there! I think you missed the alpine Dockerfile?

Comment (gnu300) [reply docker/amd64/sqlite/Dockerfile]: omg, yes of course.

**Comment by dani-garcia** [comment]
Looks good to me, thanks!

**Pull request merged**